### DISCUSSION TITLE
Is it possible to output product information to file?

### DISCUSSION THREAD

**Discussion by theo-d-smith** [jef/streetmerchant#2056] [Help]
It doesn't appear to be built in, but does anyone know of a way to output product name, price, and URL to a file (.txt,.csv,etc.) whenever a product is in stock? Or perhaps where I can find the part of the code that outputs to the cmd window. Thanks!

**Comment by PMVX** [comment]
I think what you are looking for is src/logger.ts. At least that is what I understand is responsible of putting text on the cmd window
